### ISSUE TITLE
A11y: Footer: Cookie Settings not keyboard accessible

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/clean-and-green-philly#859]
## Describe the task

Currently the "Cookie Settings" in the footer is not reachable or operable via keyboard navigation (pressing TAB). That's because it's an a element with no href. Might make more sense to be coded as a button since it triggers the cookies popup. 

## Acceptance Criteria

- [ ] You should be able to TAB to the Cookie Settings. 
- [ ] You should be able to activate Cookie Settings and make the popup show by pressing ENTER or SPACE buttons.
- [ ] The Cookie Settings should still work when using screenreader.

## Additional context

- When the cookies popup shows, the focus should move to "Can we store cookies to your browser?" but I think that's outside the scope of this ticket.

**Comment by Amberroseweeks** [comment]
Hello! I'd like to be assigned this task if it is still open. Thank you

**Comment by bacitracin** [comment]
assigned!

**Comment by nlebovits** [comment]
@bacitracin this ticket is stale. Does it need to be refined/rewritten?

**Comment by bacitracin** [comment]
@nlebovits the issue is still accurate, thanks for checking!

**Comment by Amberroseweeks** [comment]
Hi @bacitracin, Sorry for the delay, I have some time this week and next week to work on this. I will pick this up today if it's still open and ready to be worked on.

**Comment by bacitracin** [comment]
@Amberroseweeks No worries! I'll assign you

**Comment by bacitracin** [comment]
Live, so closing
